rocAL/rocAL_pybind/amd/rocal/readers.py

```diff
@@ -49,131 +33,20 @@ def file(*inputs, file_root, bytes_per_sample_hint=0, file_list='', initial_fill
-                    exit()
-        tf_meta_data = b.TFReaderDetection(Pipeline._current_pipeline._handle ,*(kwargs_pybind.values()))
-    else:
-        Pipeline._current_pipeline._reader = "TFRecordReaderClassification"
-        kwargs_pybind = {"path": path, "is_output": True, "user_key_for_label": user_feature_key_map[
-            "image/class/label"], "user_key_for_filename": user_feature_key_map["image/filename"]}
-        for key in features.keys():
-                if key not in user_feature_key_map.keys():
-                    print(
-                        "For Image Classification, ROCAL TFRecordReader needs all the following keys in the featureKeyMap:")
-                    print("image/encoded\nimage/class/label\n")
-                    exit()
-        tf_meta_data = b.TFReader(Pipeline._current_pipeline._handle ,*(kwargs_pybind.values()))
-    features["image/encoded"] = tf_meta_data
-    features["image/class/label"] = labels
-    return features
-
-def caffe(*inputs, path, bbox=False, bytes_per_sample_hint=0, image_available=True, initial_fill=1024,
-          label_available=True, lazy_init=False, num_shards=1, pad_last_batch=False, prefetch_queue_depth=1,
-          preserve=False, random_shuffle=False, read_ahead=False, seed=-1, shard_id=0, skip_cached_images=False,
-          stick_to_shard=False, tensor_init_bytes=1048576, device=None):
+def coco(*inputs,file_root='', annotations_file='', bytes_per_sample_hint=0, dump_meta_files=False, dump_meta_files_path='', file_list='', initial_fill=1024,  lazy_init=False, ltrb=False, masks=False, meta_files_path='', num_shards=1, pad_last_batch=False, prefetch_queue_depth=1,
+                 preserve=False, random_shuffle=False, ratio=False, read_ahead=False,
+                 save_img_ids=False, seed=-1, shard_id=0, shuffle_after_epoch=False, size_threshold=0.1, is_box_encoder=False, is_box_iou_matcher=False,
+                 skip_cached_images=False, skip_empty=False, stick_to_shard=False, tensor_init_bytes=1048576):
 
+    Pipeline._current_pipeline._reader = "COCOReader"
     #Output
-    bboxes = []
     labels = []
-    kwargs_pybind = {"source_path": path}
-    #Node Object
-    if (bbox == True):
-        Pipeline._current_pipeline._reader = "CaffeReaderDetection"
-        caffe_reader_meta_data = b.CaffeReaderDetection(Pipeline._current_pipeline._handle ,*(kwargs_pybind.values()))
-    else:
-        Pipeline._current_pipeline._reader = "CaffeReader"
-        caffe_reader_meta_data = b.CaffeReader(Pipeline._current_pipeline._handle ,*(kwargs_pybind.values()))
-
-    if (bbox == True):
-        return (caffe_reader_meta_data,bboxes, labels)
-    else:
-        return (caffe_reader_meta_data, labels)
-
-def caffe2(*inputs, path, bbox=False, additional_inputs=0, bytes_per_sample_hint=0, image_available=True,
```

Comment:
Multiple readers like caffe, caffe2, video, sequence_reader, tf has been removed. Please check if this is intended